_posts/fork/2021-11-28-Liquid.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: theme
+title: Liquid
+category: fork
+homepage: https://github.com/Fentaniao/Liquid
+download: https://github.com/Fentaniao/Liquid/archive/master.zip
+built-in: false
+author: Fentaniao
+thumbnail: Liquid.png
+typora-root-url: ../../
+typora-copy-images-to: ../../media/theme/Liquid
+---
+# Liquid
+
+A nice theme for Typora, inspired by Purple markdown style and Microsoft Fluent Design.
```

Comment:
Could you also add a link and thumbnail to this theme at Purple theme, so we can find your theme from Purple theme?

Reply:
I have already added the link and thumbnail to this theme at Purple theme. Could you please accept my pull request?

Reply:
> 
I have already added the link and thumbnail to this theme at Purple theme. Could you please accept my pull request?